ui/frontend/openapi-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "schemaFile": "http://127.0.0.1:8000/api/openapi.json",
+  "schemaFile": "http://127.0.0.1:8241/api/openapi.json",
```

Comment:
just double checking, this is the right port?

Reply:
yes